test/lint/lint-submodule.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+"""
+This script checks for git modules
+"""
+
+import subprocess
+import sys
+
+def get_submodules_list():
+    command = [
+            'git',
+            'submodule',
+            'status',
+            '--recursive',
+    ]
+    return subprocess.check_output(command).decode('utf-8').splitlines()
+
+def main():
+    submodules_list = get_submodules_list()
```

Comment:
```suggestion
    submodules_list = subprocess.check_output(
```

no need for a function, you can call the stdlib directly

Reply:
Sure, done !